src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -980,6 +373,12 @@ void emberAfThermostatClusterServerInitCallback(chip::EndpointId endpoint)
     // or should this just be the responsibility of the thermostat application?
 }
 
+// IMPORTANT NOTE:
+// No Side effects are permitted in emberAfThermostatClusterServerPreAttributeChangedCallback
+// If a setpoint changes is required as a result of setpoint limit change
+// it does not happen here.  It is the responsibility of the device to adjust the setpoint(s)
+// as required in emberAfThermostatClusterServerPostAttributeChangedCallback
+// limit change validation assures that there is at least 1 setpoint that will be valid
```

Comment:
this line doesn't make sense combined with the previous line

Reply:
I didn't actually write this comment; it just got relocated away from this function in the initial commit of the preset code somehow